helper_windows_test.go

```diff
@@ -116,3 +116,9 @@ func copyTestdataFileTo(src, dest string) []string {
 func singleCommandNoArgs(command string) []string {
 	return []string{command}
 }
+
+// this is opt-out so that we don't skip tests by default
+func shouldRunAdminTests() bool {
+	_, ok := os.LookupEnv("SKIP_ADMINISTRATOR_TESTS")
```

Comment:
Now we've landed #218 we should probably call this `GW_SKIP_<something>` and update the `README.md` to explain the purpose of this extra env variable. How about something like `GW_SKIP_SERVICE_INSTALLATION_TESTS`?

Reply:
I renamed it to `GW_SKIP_ADMIN_REQUIRED_TESTS` to keep with the spirit of skipping administrator privileged tests.